pkg/kubelet/cm/memorymanager/fake_memory_manager.go

```diff
@@ -32,41 +32,41 @@ type fakeManager struct {
 }
 
 func (m *fakeManager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesReady, podStatusProvider status.PodStatusProvider, containerRuntime runtimeService, initialContainers containermap.ContainerMap) error {
-	klog.Info("[fake memorymanager] Start()")
+	klog.InfoS("[fake memorymanager] Start()")
```

Comment:
```suggestion
	klog.InfoS("Start()")
```

Please remove the log prefixes from all migrations. (I won't repeat the comment on every line because that's too much typing!)